utkdigitalinitiatives/utk_ontology
# Add utk_minute_taker as a property.

## What does this Pull Request do?

This addresses the need to add a URI (https://ontology.lib.utk.edu/roles#mtk) for the property "utk_minute_taker" as was indicated in this PR - https://github.com/utkdigitalinitiatives/m3_profiles/pull/39

## What's new?

A literal property for minute taker has been added to the ontology.

## How should this be tested?

Ensure that formatting checks pass. Check the added property to make sure that the ttl accurately communicates the new role (and doesn't include any copypasta).
